### DISCUSSION TITLE
Play button grayed out with the message Cannot play this replay

### DISCUSSION THREAD

**Discussion by stefancata92** [fraxiinus/ReplayBook#254] [Troubleshooting]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by fraxiinus** [comment]
Play button is disabled when ReplayBook does not have access to a matching patch version

https://www.fraxiinus.dev/ReplayBook/troubleshooting/play-button-disabled/

**Comment by stefancata92** [comment]
Hello, from where can I get a matching patch version? I have some replays from patch 13.7

**Comment by fraxiinus** [comment]
you can wait a few days and replays.xyz will have it

https://replays.xyz/old-clients

**Comment by stefancata92** [comment]
Thank you for your response, is that website working for you? I try to get the 13.6 version but the download is not starting.

**Comment by fraxiinus** [comment]
That site is not affiliated with me.

You have to wait 40 seconds before it'll let you download. It also won't work if you're using a browser like OperaGX, turn off whatever Opera download enhancer or use something like Firefox.
